apps/accounts/constants.py

```diff
@@ -0,0 +1,13 @@
+def mail_body(host):
```

Comment:
Sorry for asking to change again but I found a more decent way of doing this. It would be nice if we can have a directory `apps/accounts/templates/email/` where we store all the email related body and messages. It is very similar to how django-allauth structures the directory. See https://github.com/pennersr/django-allauth/tree/master/allauth/templates/account/email for more details. 

Please let me know if you didn't understand something.